src/librustc_error_codes/error_codes/E0724.md

```diff
@@ -0,0 +1,25 @@
+`#[ffi_returns_twice]` was used on non-foreign function.
+
+Erroneous code example:
+
+```compile_fail,E0724
+#![feature(ffi_returns_twice)]
+#![crate_type = "lib"]
+
+#[ffi_returns_twice] // error!
+pub fn foo() {}
+```
+
+`#[ffi_returns_twice]` can only be used on foreign functions.
+For example, we might resolve the previous example by declaring
+inside an `extern` block annotated with a `#[link]` attribute containing
```

Comment:
You’ll need to update this sentence to reflect the changed example that follows.